rcl/include/rcl/subscription.h

```diff
@@ -69,25 +69,29 @@ rcl_get_zero_initialized_subscription(void);
  * When the user defines a ROS message, code is generated which provides the
  * required rosidl_message_type_support_t object.
  * This object can be obtained using a language appropriate mechanism.
- * \TODO(wjwwood) probably should talk about this once and link to it instead
- * For C this macro can be used (using std_msgs/String as an example):
+ * \todo TODO(wjwwood) write these instructions once and link to it instead
```

Comment:
Yes, the `\todo` causes doxygen to render it, but the `TODO` is something that can be found by our linters and editors automatically. Doxygen will not render `\TODO` and my editor will not highlight `\todo` only.